index.html

```diff
@@ -49,5 +47,183 @@ <h1 class="intro__heading">
+                <img src="./images/SnapshootPortfolio3.png" alt="image of a project" class="img__card">
+                <H2 class="desc__project">Multi-Post Stories</H2>
+              <ul class="project__info--cont">
+                <li class="project__info--title">canopy</li>
+              <li class="project__info--dot">
+                    <i class="fas fa-circle color__circle"></i>
+                </li>
+                <li class="project__info--author">Back End Dev</li>
+                 <li class="project__info--dot">
+                  <i class="fas fa-circle color__circle"></i>
+                 </li>
+                <li class="project__info--date">2015</li>
+              </ul>
+            <p class="project__desc--text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="list__lng--btn">
+                <li class="lng__btn--cont"> <button type="button" class="lng__btn">html</button></li>
+                <li class="lng__btn--cont"> <button type="button" class="lng__btn">css</button></li>
+                <li class="lng__btn--cont"> <button type="button" class="lng__btn">javascript</button></li>
+            </ul>
+             <button type="button" class="link__btn--project">See Project</button>
+             </figcaption>
+        </figure>
+
+    </section>
+    
+    <section class="about">
+     <div class="intro intro__resume">
+            <h1 class="intro__heading">
+                About Myself
+            </h1>
+            <p class="intro__text">
+                I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! If you like what you 
+                see and have a project you need coded, don’t hestiate to contact me.</p>
+             <a href="#" class="intro__contact">let's connect</a>
+            <ul class="intro__social--cont">
+                
+                <li class="social__item"> <a href="#" class="link__social">  <i class="fab fa-github-square"></i> </a> </li>
+                <li class="social__item"> <a href="#" class="link__social">  <i class="fab fa-linkedin-in"></i> </a> </li>
+                <li class="social__item"> <a href="#" class="link__social">  <i class="far fa-hand-peace"></i> </a> </li>
+                <li class="social__item"> <a href="#" class="link__social">  <i class="fab fa-twitter"></i> </a>   </li>
+                <li class="social__item"> <a href="#" class="link__social">  <i class="fab fa-medium-m"></i>   </a>   </li>
+
+            </ul>
+             
+            <button type="button" class="link__btn--resume">Get my resume</button>
+        </div>
+       <div class="skills__cont">
+           <ul class="list__skills--cont">
+               <li class="cont__skill">
+                   <p class="name__skill">Languages</p>
+                   <i class="fas fa-chevron-down fa-xs icon__chevron"></i>
```

Comment:
- [ ] **[OPTIONAL]** Your class names are quite unconventional. The use of underscores for class names was normal in the past but now, you should use hyphens instead. It is suggested that using underscores can make working with JavaScript more difficult than it needs to. Your code would be more readable too to a modern developer. You can refer to this [StackOverflow answer](https://stackoverflow.com/a/7561434)

For example:
- `link__btn--resume` can be changed to `link-btn-resume`
- `skills__cont` can be `skills-cont`